page-objects/pages/coursePages/CRSSEditCopyCoursePage.js

```diff
@@ -29,10 +30,16 @@ async function setNewCourseDescription(courseDescription) {
 	await waitHelpers.waitAndSetValue(courseDescriptionInput, courseDescription);
 }
 
+async function checkThatTeacherIsVisible(courseTeacher, expectedResult) {
```

Comment:
Good practice: Include error message like 
const msg = `Teacher with name ${teacherName} is not visible on the list`;
	const resultMsg = `List of teacherNames:  ${teacherNames}`;
	return expect(teacherNames, msg + resultMsg).to.include(courseTeacher);